merge_sort.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import unicode_literals
+from time import time
+
+
+def merge_sort(init_list):
+    if len(init_list) <= 1:
+        return init_list
+    list_left = init_list[:len(init_list) / 2]
+    list_right = init_list[len(init_list) / 2:]
```

Comment:
Watch out for true division bugs.  You never want a float here in your index, but you'll get one for odd-length lists in Python 3.